delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/RedundantInherited.html

```diff
@@ -0,0 +1,7 @@
+<h2>Why is this an issue?</h2>
+<p>
+  It is possible to use a bare inherited statement without the method overriding a parent's.
+  This is misleading and makes the code more confusing.
```

Comment:
This description could be a bit more fleshed out, and also have better clarity.

For example, using the term "overriding" may lead people to think of `virtual` and `override`, which are **adjacent** concerns to `inherited` but actually irrelevant to this rule (since `inherited` still works for non-virtual methods, yeah?)

Reply:
When I wrote the description, I had not considered the overloaded terms for that non-virtual methods. In light of that, should I change the test names to avoid using `Overriding` as none of them actually use `override`?

I will give the description another go.

Reply:
No worries about the test names, it's a reasonable descriptor for the code and adding or removing the `override` keyword won't change their result.